### ISSUE TITLE
very long electron steps

### ISSUE THREAD

**Issue by jw3126** [nrc-cnrc/EGSnrc#387]
I am running a simulation of a chamber in a waterphantom using `egs_chamber`. I use cross section enhancement in a box around the chamber. This results in some of the electron steps being extremely long. At least thats how I interpret the long red lines in the attached `egs_view` image:
![longelectrontracks](https://user-images.githubusercontent.com/7261506/33421072-290b26d0-d5b1-11e7-909a-88b3e41067e8.PNG)

**Comment by ftessier** [comment]
Indeed, variance reduction techniques play tricks with the particle stack, which in some cases confuses the particle track scoring object. Tracks can only be trusted when all variance reduction is turned off. It would be a nice improvement to resolve this issue, in order to visualize variance reduction techniques properly.

**Comment by jw3126** [comment]
Thanks for looking into this! So this is only an issue with the track scoring? For example energy deposition should be correct?

**Comment by rtownson** [comment]
Yes the energy deposition will be correct! Just the tracks are wrong.

**Comment by jw3126** [comment]
Thanks for the confirmation!
